.gitignore

```diff
@@ -3,6 +3,7 @@ node_modules/
 npm-debug.log
 report/
 clover.xml
+public/mix-manifest.json
 
```

Comment:
shouldn't there also be a `mix-manifest.json`?

Reply:
Yes, there should be a `mix-manifest.json`. But not in the repo as it is regenerated everytime `npm run x` is running.